pkg/datapath/loader/base.go

```diff
@@ -162,6 +162,20 @@ func cleanIngressQdisc(devices []string) error {
 	return nil
 }
 
+// cleanCallsMaps is used to remove any pinned map matching mapName from bpf.TCGlobalsPath().
+func cleanCallsMaps(mapName string) error {
```

Comment:
I would rename the parameter to something like `mapNamePattern`, otherwise its unclear that glob patterns are allowed.